### DISCUSSION TITLE
Zero123++

### DISCUSSION THREAD

**Discussion by Disty0** [vladmandic/sdnext#2421] [General]
For Zero123++;

Set "Execution backend" to "diffusers" and fully restart.
Set "Custom Diffusers pipeline to use" to `Disty0/zero123plus-pipeline`
Set "Diffusers pipeline" to "Custom Diffusers Pipeline".
Then download `sudo-ai/zero123plus-v1.1` from Models/Huggingface tab.
(Enter `Disty0/zero123plus-pipeline` as the custom pipeline in Models/Huggingface tab too.)

Use Img2Img tab with these settings:
Sampler: DEIS or DPM++ 2M (With Karras)
Steps: 75 (For anime. Realistic can use less steps.)
CFG: 4
Image CFG: 1

You can use simple prompts but detailing it too much can cause nightmare fuels.
Negatives will be ignored.

![image](https://github.com/vladmandic/automatic/assets/47277141/6b49d0f3-2740-48d1-845e-1a0e2d308f68)

**Comment by vladmandic** [comment]
Bit updated steps:

Prepare:
1. Settings -> Execution backend: diffusers
2. Models -> Huggingface: download `sudo-ai/zero123plus-v1.1`
3. Restart

Activate:
1. Settings -> Diffusers pipeline: Custom Diffusers Pipeline
2. Settings -> Load custom Diffusers pipeline: `Disty0/zero123plus-pipeline`
3. Load model: `sudo-ai/zero123plus-v1.1`

Deactivate:
1. Settings -> Diffusers pipeline: Autodetect
2. Load any standard model

**Comment by Ratinod** [comment]
~`ERROR    Processing: Inputting embeddings not supported for this pipeline. Please pass an image.`~
~Some help please?~
~Or perhaps a strictly dev branch is required?~
***
Upd.
Dev Branch is required. It works now.

**Reply by Disty0** [reply]
Dev branch merged to master today. Dev branch isn't needed anymore.
